eth2/state_processing/src/per_epoch_processing/process_slashings.rs

```diff
@@ -13,12 +14,17 @@ pub fn process_slashings<T: EthSpec>(
 
     for (index, validator) in state.validators.iter().enumerate() {
         if validator.slashed
-            && epoch + T::EpochsPerSlashingsVector::to_u64() / 2 == validator.withdrawable_epoch
+            && epoch + T::EpochsPerSlashingsVector::to_u64().wrapping_div(2)
```

Comment:
Double checking that this should be `wrapping_div`

Reply:
I used wrapping div, because dividing by 2 is always safe, but it's a premature optimisation, so I've switched it to `safe_div`. The compiler will work it out